spec/classes/letsencrypt_spec.rb

```diff
@@ -19,13 +19,13 @@
 
           epel = facts[:osfamily] == 'RedHat' && facts[:operatingsystem] != 'Fedora'
 
-          it 'contains File[/usr/local/sbin/letsencrypt-domain-validation]' do
+          it 'contains the domain validation script' do
             is_expected.to contain_file('/usr/local/sbin/letsencrypt-domain-validation').
               with_ensure('file').
               with_owner('root').
               with_group('root').
               with_mode('0500').
-              with_source('puppet:///modules/letsencrypt/domain-validation.sh')
+              with_content(%r{result=}) # TODO: what should be check?
```

Comment:
Probably sufficient to check for the shebang.

Reply:
done: https://github.com/voxpupuli/puppet-letsencrypt/pull/275/commits/cacb110a18d8f8fcd52f2897188f87caabef2c4c